irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -21,8 +21,9 @@
 
 #include "model/execution/command_executor_factory.hpp"
 
-#include "ametsuchi/wsv_command.hpp"
 #include "model/sha3_hash.hpp"
+#include "backend/protobuf/from_old_model.hpp"
+#include "interfaces/iroha_internal/block.hpp"
```

Comment:
This is included by `from_old_model`.

Reply:
We may move to the forward declaration, so methods can be unavailable

Reply:
Header can be added when such move will be done.